src/test/integration/translation-validation.test.ts

```
@@ -0,0 +1,152 @@
+  ];
+
+  function validateTranslationUniqueness(articles: typeof mockArticles) {
+    // Grouper par langue
+    const postsByLang = new Map<string, typeof mockArticles>();
+    
+    articles.forEach(post => {
+      const lang = post.data.lang;
+      if (!postsByLang.has(lang)) {
+        postsByLang.set(lang, []);
+      }
+      postsByLang.get(lang)!.push(post);
```

Biome (1.9.4)

[error] Forbidden non-null assertion.

Unsafe fix: Replace with optional chain operator ?. This operator includes runtime checks, so it is safer than the compile-only non-null assertion operator


(lint/style/noNonNullAssertion)